src/utils/const/errorMessages.js

```diff
@@ -0,0 +1 @@
+export const ERROR_MESSAGE_TASK_TITLE = "Please enter valid a task";
```

Comment:
is this a task validation error message?

Reply:
Yes vai. Changed const name to more understandable name. Thank you.